.devcontainer/Dockerfile

```diff
@@ -42,6 +42,18 @@ RUN useradd -s /bin/bash -u $USER_UID -g $USER_GID -G docker -m $USERNAME
 RUN echo $USERNAME ALL=\(root\) NOPASSWD:ALL > /etc/sudoers.d/$USERNAME
 RUN chmod 0440 /etc/sudoers.d/$USERNAME
 
+# Python 3.9 and PIP
```

Comment:
should we place this in the chip-vscode docker image instead?

Reply:
I see this is already done. Why are we updathing this as well?

This is `FROM connectedhomeip/chip-build-vscode:${BUILD_VERSION}`

Reply:
I added it here so it can be built with in the dev container as well, but I can remove it.

Reply:
Anything put into chip-build-vscode will be available in the devcontainer since the devcontainer build starts from that image. That's what Andrei is saying.